Foundation/NSString+Private.h

```diff
@@ -0,0 +1,8 @@
+#import <Foundation/Foundation.h>
+
+@interface NSString (Private)
+
++ (NSString *)stringWithUnichar:(unsigned int)aChar;
++ (NSString *)_stringWithUnichar:(unsigned int)aChar;
```

Comment:
Interesting, any idea what the difference is between underscore and no underscore? Would one be more preferable over the other (e.g. one just calls the other method) or do they have different implementations?

Reply:
Yeah, one of them calls the other. I will remove the underscored version.